ZeeHomeScreen/Classes/Components/BasicContainerCell/UniversalCollectionViewHeaderFooterView.h

```diff
@@ -5,18 +5,22 @@
 //  Created by Miri on 28/01/2020.
 //
 
-
 @import UIKit;
 
 @protocol ComponentProtocol;
 @protocol UniversalCollectionViewHeaderFooterViewDelegate;
 
 @interface UniversalCollectionViewHeaderFooterView : UICollectionReusableView
-@property (nonatomic, strong) UIViewController <ComponentProtocol> *componentViewController;
-@property (nonatomic, weak)id <UniversalCollectionViewHeaderFooterViewDelegate>delegate;
+
+@property (nonatomic, weak) UIViewController<ComponentProtocol> *componentViewController;
```

Comment:
componentViewController is already being held by childViewControllers, so there is no need to introduce yet another strong reference

Reply:
was manifesting on scrolling HomeScreen vertically.